web/src/main/java/se/inera/intyg/webcert/web/service/arende/ArendeServiceImpl.java

```diff
@@ -744,6 +787,27 @@ private void verifyEnhetsAuth(String enhetsId, boolean isReadOnlyOperation) {
 
     }
 
+    private boolean isNotDuplicateAnswer(AnsweredWithIntyg answerWithCertificate, List<AnsweredWithIntyg> answeredWithIntygList) {
+        return answeredWithIntygList.stream()
+            .noneMatch(answeredWithIntyg -> answeredWithIntyg.getIntygsId().equalsIgnoreCase(answerWithCertificate.getIntygsId()));
+    }
+
+    private AnsweredWithIntyg checkAnsweredFromCertificateOutsideWebcert(String certificateId) {
```

Comment:
Ska metoden heta check?

Reply:
Förslag på namn? :-)

Reply:
getAnsweredFromCertificateOutsideWebcert?

Reply:
Varför inte! Jag tror nog att jag först bara tänkte kontrollera om den var besvarade eller inte av ett intyg utanför Webcert och därav det tidigare namnet. Men den både kontrollerar och i så fall returnerar. Ändrar enligt förslag!

Reply:
Fixat. Checkar in ändringar samtidigt när du är klar med reviewn.